src/domains/login/common/components/NaverLogin.jsx

```
@@ -1,19 +1,30 @@
 import navericon from '../../../../assets/icons/common/naverIcon.svg'
+import navericon_lg from '../../../../assets/icons/common/naverIcon_lg.png'
+import useIsMobile from '../../../../hooks/header/useIsMobile'
 
 const NaverLogin = () => {
+  const isMobile = useIsMobile()
```

ESLint

[error] 'isMobile' is assigned a value but never used.

(no-unused-vars)